models/Tool/Email/Log.php

```diff
@@ -320,20 +329,38 @@ public function getEmailLogExistsText(): int
 
     /**
      * Returns the filename of the html log
+     * @deprecated since Pimcore 12 use getHtmlLogFilenameWithDate() instead
```

Comment:
the replacement is `private` though
https://github.com/pimcore/pimcore/pull/18792/files#diff-16574d90a9d6e1e8af9bb3c2a93854fbd8ed0e34b1c0f923e6d1a9b147bc8e1bR339

Reply:
we would just need to keep `getHtmlLogFilename` in future, currently considering `getHtmlLogFilenameWithDate` but in v13, we stick with `getHtmlLogFilename` only performing what the `withDate` do.

Reply:
@kingjia90 Yes because the class is internal so this shouldn't be a problem

Reply:
ahh ok, so it's for us internally to adapt, then all good, maybe we shouldn't need this deprecation line but won't hurt either